docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -443,6 +443,38 @@ paths:
           description: Group not found
           schema:
             $ref: '#/definitions/Error'
+    post:
+      tags:
+        - groups
+      summary: Add a member to a group
+      operationId: addGroupMember
+      parameters:
+        - name: id
+          in: path
+          description: Public ID of the group
+          required: true
+          type: string
+        - name: user
```

Comment:
You mention in another comment that the `user` param seems misnamed, and I agree. The user-fetching service you use in the view will accept either a `userid` (which combines username and authority) or a separate `username` and `authority` param. For simplicity sake, I'd suggest naming this `userid`, and adding documentation about the syntax of a userid (`acct:username@authority`).

Reply:
(I respect that this conflicts with the `user`-named parameter in the `remove-group-member` documentation, but I think that endpoint is a little peculiar :) )